packages/@okta/vuepress-site/docs/guides/configure-signon-policy/main/index.md

```diff
@@ -75,73 +76,107 @@ The following are step-by-step instructions to configure a global session policy
-
-7. For this use case example, select **8 hours** for **Session Expires After**. Use these fields to specify the maximum idle time before an authentication prompt is triggered. The maximum allowed time for this option is 90 days. This isn't the total connection time. This is idle time before users see a countdown timer at the 5-minute mark of remaining session time.
+1. Click **Set time limit** and leave the default time settings.
 
     > **Note:** You can also set the [maximum session lifetime value](/docs/reference/api/policy/#signon-session-object) using the Okta APIs. If you previously set this value using the API, you can't exceed that maximum in the Admin Console. Setting a value over the API maximum results in an error.
 
-8. Click **Create Rule**.
+1. Click **Create Rule**.
 
 > **Note:** After you create a policy, you must close all active sessions for the new policy to take effect.
 
 ## Prompt for an additional factor when a user is outside the US
 
-You may want all default Okta users to provide a password, but you want all Okta users outside of the United States to provide both a password and another factor to access your app. You can use the default authentication policy’s catch-all rule that challenges all users to provide a password. Then, create another rule that challenges all users not in the United States to provide both a password and another factor each time that they sign in.
+You may want a rule that requires all default Okta users to provide a password. But you also want all Okta users outside of the United States to provide both a password and another factor to access your app. You can use the default authentication policy’s catch-all rule that challenges all users to provide a password. Then, create another rule that challenges all users not in the United States to provide both a password and another factor each time that they sign in.
+
+Configure another rule for the default authentication policy to prompt a user for an additional factor when the user is outside of the United States.
+
+> **Note:**  You can add as many rules to the default authentication policy that you want. But remember that changes to the default authentication policy are applied to all new apps because it's a shared app policy.
```

Comment:
```suggestion
> **Note:**  You can add as many rules to the default authentication policy as you want. But remember that changes to the default authentication policy are applied to all new apps because it's a shared app policy.
```